### PR TITLE
Artifact Balancing

### PR DESCRIPTION
**Pull request by MemeWizard**
Changelog:
- Changed artifacts balance.
- Changed libram slot into relic slot.

### CODE REVIEW

File: common/artifacts/000_wc_artifacts.txt [hunk 1]
```diff
@@ -526,8 +526,8 @@ runeblade_blood_2 = {
 	monthly_character_prestige = 0.5
 	
 	command_modifier = {
-		morale_defence = 0.05
 		defence = 0.05
+		morale_defence = 0.025
```
[the remark this answers is not in the document]
Comment (arithon) [reply common/artifacts/000_wc_artifacts.txt]: maybe revert change to morale defence and reduce defence then?

**Comment by zumbak04** [comment]
> runeblades stat grow is not linear, reee @zumbak04

@MemeWizard fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/864/commits/af310ecf6b4a9b930e0a18a9105cbd6d961a4f04 , REE!

File: common/artifacts/000_wc_artifacts.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (zumbak04) [reply common/artifacts/000_wc_artifacts.txt]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/864/commits/af310ecf6b4a9b930e0a18a9105cbd6d961a4f04 .

[the remark this answers is not in the document]
Comment (MemeWizard) [reply common/artifacts/000_wc_artifacts.txt]: Not fixed

Untitled

[the remark this answers is not in the document]
Comment (zumbak04) [reply common/artifacts/000_wc_artifacts.txt]: Fixed in https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/864/commits/e7495c3ba39abe6492de8c2043597c03eec5d82c .

**Pull request merged**